Sources/Core/Private/Resolver.swift

```diff
@@ -225,13 +251,14 @@ class Resolver {
     func initialObject() -> Any? {
       if let obj = usingObject {
         log(.verbose, msg: "Use object: \(obj)")
+        setNSResolverIfNeeded(to: obj)
```

Comment:
move to a higher level - after call initialObject

Reply:
Fixed.